### PR TITLE
fix: added git release info.

### PR DESCRIPTION
**Pull request by zhyd1997**
Fix: https://github.com/jhipster/jhipster-lite/issues/8026

### CODE REVIEW

**Comment by zhyd1997** [comment]
Hi @pascalgrimaud 

How about this, do we need to include git release time? 👀 

Screenshot 2023-12-08 at 19 36 40

**Comment by zhyd1997** [comment]
and I am not familiar with Java, so do I need to add `git.build` info to these files? 👀 

<details>
  <summary>https://github.com/search?q=repo%3Ajhipster%2Fjhipster-lite+%22git-branch%22&type=code</summary>
  
  Screenshot 2023-12-08 at 19 50 33
</details>

File: pom.xml [hunk 1]
```diff
@@ -577,6 +577,7 @@
               <includeOnlyProperty>^git.commit.id.abbrev$</includeOnlyProperty>
               <includeOnlyProperty>^git.commit.id.describe$</includeOnlyProperty>
               <includeOnlyProperty>^git.branch$</includeOnlyProperty>
+              <includeOnlyProperty>^git.build.(time|version)$</includeOnlyProperty>
```
Comment (zhyd1997) [inline pom.xml:580]: https://github.com/git-commit-id/git-commit-id-maven-plugin/blob/master/docs/using-the-plugin.md

**Comment by pascalgrimaud** [comment]
Indeed, I think it would be better.
You need to update Header.component.ts and Header.spec.ts to use the build.version instead of the current id.describe

See https://github.com/jhipster/jhipster-lite/blob/main/src/main/webapp/app/common/primary/header/Header.component.ts#L23

**Comment by pascalgrimaud** [comment]
About Consul, yes it could be updated, but it's better to do it in another PR, not in this one

**Comment by zhyd1997** [comment]
Morning @pascalgrimaud 

This is the result:

Screenshot 2023-12-09 at 09 46 54

**Comment by pascalgrimaud** [comment]
thanks a lot @zhyd1997 : let's merge this

**Pull request merged**